CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 
 - Introduced a new  `%ffi` extension (*experimental* - not for production use!) that provides a more robust mechanism for JavaScript function interoperation by considering function arity in type constraints. This enhancement improves safety when dealing with JavaScript functions by enforcing type constraints based on the arity of the function. https://github.com/rescript-lang/rescript-compiler/pull/6251
 - Extended untagged variants with function types. https://github.com/rescript-lang/rescript-compiler/pull/6279
+- Untagged variants: consider regexp as an object type. https://github.com/rescript-lang/rescript-compiler/pull/6296
```

Comment:
This needs to go to beta.3 actually, I will do the PR for preparing beta.3 after I have got the "publish to npm on tag build" stuff working.

Reply:
I'll rebase.